free-1.md

```diff
@@ -5,47 +5,63 @@
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+lets say the user wants to store information they may use a key word such as store or save ... The laguange would see the keyword and automaticcly go through the process of setting up a data base. Lets also say they wanted to save a book and maybe we have a keyword for book, this would allow the system to start a category within the dabase for books with additional automation. once the users request is put through it would comunicate back with a success or fail message where the fail message may be because of an operational issue or maybe becuase the language used by the user did not mach the specifations of the DSL.
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy for the user to put, gather, and edit info within a database through a limited key word language. This shoud be possible but may be difficult to program effectivly. I dont believe anything within this project would be impossible but the natural langauge component to actual process in real time with automation may be more difficult. In the sence that when a keyword is detected the program is basically writing a file for a database auomaticaly instead of the user or programmer. 
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I do not think there is a language for this specifically. There are databases and storage tools such as notes, zotero, online databases, and much more. But functionality wise there are no laguanges that allow for new info to be stored upon user request affectivly and easily with minimal user actions. 
+
```

Comment:
I guess I'm curious about some more specifics about why a database is a good place for notes, beyond that it would be all in one space. I feel like all those independent tools have different functionalities because of the different types of data it stores.